swagger/cosmos.yaml

```diff
@@ -0,0 +1,938 @@
+swagger: '2.0'
```

Comment:
Can we have the spec definitions in an `openapi` directory? As it's the new name for them.